.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
```

Comment:
But I don't feel like it's a strong enough win for you, if it must be enforced like this.

I will remove it from this PR since it's not really the main focus.